synctl

```diff
@@ -2320,6 +2380,8 @@ class ParseParameter:
             '--host', type=str, help='set hostname')
         self.parser_delete.add_argument(
             '--token', type=str, help='set token')
+        self.parser_delete.add_argument(
+            '--key', type=str, help='delete credential with credential name')
```

Comment:
remove `--key` here, we use same syntax with `synctl delete lo <location-id1> ...`, for creds, the command should be `synctl delete cred <cred-name> <cred-name2> ...`, get_args.id can get multiple key name. When get_args.delete_type is SYN_CRED, get_args.id should not be none, and at least one key-name. you can refer this pop_instance.delete_synthetic_locations(get_args.id).